spark/src/test/scala/org/apache/spark/sql/comet/CometPlanStabilitySuite.scala

```diff
@@ -278,7 +278,6 @@ trait CometPlanStabilitySuite extends DisableAdaptiveExecutionSuite with TPCDSBa
       CometConf.COMET_SHUFFLE_FALLBACK_TO_COLUMNAR.key -> "true",
       CometConf.COMET_EXEC_SORT_MERGE_JOIN_WITH_JOIN_FILTER_ENABLED.key -> "true",
       CometConf.COMET_CAST_ALLOW_INCOMPATIBLE.key -> "true", // needed for v1.4/q9, v1.4/q44, v2.7.0/q6, v2.7.0/q64
-      "spark.sql.readSideCharPadding" -> "false",
```

Comment:
The one code change is to remove this config

Reply:
Why did we ever have this, I wonder?

Reply:
My guess is that at some point we fell back to Spark because we could not handle expressions such as `staticinvoke(class org.apache.spark.sql.catalyst.util.CharVarcharCodegenUtils, StringType, readSidePadding, cd_education_status#25, 20, true, false, true) AS cd_education_status#33,`